Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionBookmarks.h

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2025  All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <WebKit/WKFoundation.h>
+
+
+@class WKWebExtensionContext;
+@protocol _WKWebExtensionBookmark;
+
+WK_HEADER_AUDIT_BEGIN(nullability, sendability)
+
+/*!
+ @abstract Constants used by ``_WKWebExtensionBookmark`` to indicate the type of a bookmark node.
+ @constant _WKWebExtensionBookmarkTypeBookmark  Indicates the node is a bookmark with a URL.
+ @constant _WKWebExtensionBookmarkTypeFolder  Indicates the node is a folder that can contain other bookmarks or folders.
+ @constant _WKWebExtensionBookmarkTypeSeparator  Indicates the node is a separator.
+ */
+typedef NS_ENUM(NSInteger, _WKWebExtensionBookmarkType) {
+    _WKWebExtensionBookmarkTypeBookmark,
+    _WKWebExtensionBookmarkTypeFolder,
+} NS_SWIFT_NAME(_WKWebExtension.BookmarkType) WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_VISIONOS_TBA));
+
+/*! @abstract A class conforming to the ``_WKWebExtensionBookmark`` protocol represents a single bookmark node (a bookmark, folder, or separator) to web extensions. */
```

Comment:
This comment says separator, but it doesn't look like we have any support for separators. Is this future work, or should the comment be removed?